### DISCUSSION TITLE
VisionOS

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#331] [General]
Today, Apple announced its new Vision Pro machine, something that combines VR and AR technology. So far, it's been shown to have the capability to be like a computer that doesn't require screens. This makes me wonder, what would ToonSquid be like in VisionOS?

**Comment by Cat-Butler** [comment]
Cynical curmudgeon mode on (no offense meant to you, @DWrightAnimation ):

From what I can see, exactly like the current version but projected in mid-air through the VR/AR headset? 

I think there are important areas where VR and AR can add significant benefits (virtual anatomical applications for health care education, engineering and mechanical design and training, architectural preview), but I do not think ToonSquid is currently one of them.
However, I may well be proven wrong by more imaginative people than myself.

Of course, if you are really keen on this type of thing, you can [start practising today](https://www.slimlinewarehouse.com.au/212/ipad-floor-stands). 😹
